### ISSUE TITLE
Question about the RGB / RGBA output format

### ISSUE THREAD

**Issue by JingyeChen** [cvlab-columbia/pix2gestalt#11]
Thanks for your excellent work and it indeed inspired me a lot! I am wondering is it possible to generate completed images in RGBA format for easier segmentation?

**Comment by egeozguroglu** [comment]
Thanks! Yes, it is practically possible to output RGBA by adding an output channel to the latent diffusion architecture, initializing it from scratch, followed by our fine-tuning process. To best match the image-to-image Stable Diffusion pre-training (where the output is RGB), however, we did not do this. The trade-off is between slightly better output masks but worse priors & out-of-distribution generalization. Instead, our model generates the whole object & only the whole object (ie without any pixels belonging to the occluders & background) on a white background, which we simply [threshold](https://github.com/cvlab-columbia/pix2gestalt/blob/1726a9d3e0f3e95601121680939388cfa8d19dfa/pix2gestalt/app.py#L152) or apply [matting](https://github.com/cvlab-columbia/pix2gestalt/blob/1726a9d3e0f3e95601121680939388cfa8d19dfa/pix2gestalt/app.py#L155) to retrieve the predicted amodal mask in binary form.
